### DISCUSSION TITLE
Extending the example

### DISCUSSION THREAD

**Discussion by markacola** [jakeswenson/notion#55] [Ideas]
It would be neat to have a nice example to follow beyond the existing tests, and the current example seems to be missing some implementation details.

Let me know if you'd like me to put something together.

**Comment by jakeswenson** [comment]
That'd be awesome, do you have an idea? our would you think about completing the todo list one?

**Reply by markacola** [reply]
I was thinking of keeping the current example much the same and adding the implementation in the current todo sections.

Possibly removing the message passing to simplify the example and focus on just the lib usage.

**Comment by incrypto32** [comment]
@markacola I echo your thoughts on improving the example. Would love to contribute to it. And maybe even create more examples.
